arex-storage-web-api/src/main/java/com/arextest/storage/service/handler/mocker/coverage/CoverageMockerHandler.java

```diff
@@ -83,7 +88,8 @@ public void handleOnRecordSaving(Mocker coverageMocker) {
     } else if (CaseSendScene.MIXED_NORMAL.name().equals(scheduleSendScene) &&
         handlerSwitch.allowReplayTask(appId)) {
       scenePoolProvider = scenePoolFactory.getProvider(ScenePoolFactory.REPLAY_SCENE_POOL);
-      task = new ReplayTask(scenePoolProvider, updateCaseStatusService, coverageMocker);
+      task = new ReplayTask(scenePoolProvider, updateCaseStatusService, coverageMocker, cacheProvider,
```

Comment:
non-static private class can use the resource of parent class directly, no need to pass cacheProvider and appConfig into the child class